config/jobs/kubernetes/sig-release/release-branch-jobs/1.22.yaml

```diff
@@ -646,6 +699,60 @@ periodics:
       image: gcr.io/k8s-testimages/kubekins-e2e:v20210721-2b77449-1.22
       name: ""
       resources: {}
+- annotations:
+    testgrid-dashboards: google-windows, sig-windows-gce, sig-windows-master-release,
```

Comment:
`sig-windows-master-release` dashboard would be for 1.23 jobs+ there is a `sig-windows-1.22-release` dashboard

Reply:
Thanks James for pointing out! Will update all accordingly.